src/components/NavBar/index.jsx

```diff
@@ -7,11 +7,11 @@ import logoWhite from 'assets/logo_white.svg';
 import menu from 'assets/icons/menu.svg';
 
 const links = [
-  // { text: 'Schedule', to: '/schedule' },
-  // { text: 'Maps', to: '/maps' },
-  // { text: 'Prizes', to: '/prizes' },
-  // { text: 'Travel', to: '/travel' },
-  // { text: 'Projects', to: '/projects' },
+  { text: 'Schedule', to: '/schedule' },
```

Comment:
Make sure you merge with dev. The only links that should be uncommented are /schedule, /maps, and /mentors